js/UrbanAirship.js

```diff
@@ -402,6 +402,24 @@ class UrbanAirship {
   static isQuietTimeEnabled(): Promise<boolean> {
     return UrbanAirshipModule.isUserNotificationsEnabled();
   }
+
+  /**
+   * Sets the badge number for iOS.
+   *
+   * @param {number} badgeNumber specified badge to set.
+   */
+  static setBadgeNumber(badgeNumber: number) {
+    UrbanAirshipModule.setBadgeNumber(badgeNumber);
```

Comment:
Would the following make sense? 
```
if (Platform.OS == 'ios') {
      UrbanAirshipModule.setBadgeNumber(badgeNumber);
} else {
      console.log('This feature is not supported on this platform.')
 }
```
Or is this something that should be handled within app code? 

```
if (Platform.OS == 'ios') {
  UrbanAirship.setBadgeNumber(1);
}
```